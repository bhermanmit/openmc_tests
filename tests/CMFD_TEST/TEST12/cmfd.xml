<?xml version="1.0"?>
<cmfd>

  <mesh>
    <origin> -10 -10 -1 </origin>
    <dimension> 10 10 1 </dimension>
    <width> 2.0 2.0 2.0 </width>
    <albedo> 1.0 0.0 1.0 0.0 1.0 1.0 </albedo>
<!--    <map> 
           2 2 2 2 2 2 2 2 2 2
           2 2 2 2 2 2 2 2 2 2
           2 2 2 2 2 2 2 2 2 2
           2 2 2 2 2 2 2 2 2 2
           2 2 2 2 2 2 2 2 2 2
           2 2 2 2 2 2 2 2 2 2
           2 2 2 2 2 2 2 2 2 2
           2 2 2 2 2 2 2 2 2 2
           2 2 2 2 2 2 2 2 2 2
           2 2 2 2 2 2 2 2 2 2
    </map> -->
  </mesh>

</cmfd>
