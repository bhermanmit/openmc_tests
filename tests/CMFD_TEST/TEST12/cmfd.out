MESH (1. 1, 1)  GROUP 1           8.91409E-05
MESH (2. 1, 1)  GROUP 1           -2.19179E-04
MESH (3. 1, 1)  GROUP 1           1.89479E-04
MESH (4. 1, 1)  GROUP 1           -3.07112E-04
MESH (5. 1, 1)  GROUP 1           7.13270E-05
MESH (6. 1, 1)  GROUP 1           -2.66067E-04
MESH (7. 1, 1)  GROUP 1           -1.75298E-04
MESH (8. 1, 1)  GROUP 1           -1.04119E-04
MESH (9. 1, 1)  GROUP 1           -5.69916E-05
MESH (10. 1, 1)  GROUP 1          -1.38778E-17
MESH (1. 2, 1)  GROUP 1           5.74646E-05
MESH (2. 2, 1)  GROUP 1           3.17750E-04
MESH (3. 2, 1)  GROUP 1           5.90655E-05
MESH (4. 2, 1)  GROUP 1           3.23557E-04
MESH (5. 2, 1)  GROUP 1           -1.54519E-04
MESH (6. 2, 1)  GROUP 1           1.43997E-04
MESH (7. 2, 1)  GROUP 1           6.18576E-05
MESH (8. 2, 1)  GROUP 1           -1.77988E-04
MESH (9. 2, 1)  GROUP 1           1.32149E-04
MESH (10. 2, 1)  GROUP 1          4.16334E-17
MESH (1. 3, 1)  GROUP 1           4.18513E-05
MESH (2. 3, 1)  GROUP 1           2.58486E-05
MESH (3. 3, 1)  GROUP 1           4.91012E-05
MESH (4. 3, 1)  GROUP 1           2.27435E-04
MESH (5. 3, 1)  GROUP 1           3.40576E-04
MESH (6. 3, 1)  GROUP 1           7.83557E-05
MESH (7. 3, 1)  GROUP 1           2.72552E-04
MESH (8. 3, 1)  GROUP 1           -8.32667E-17
MESH (9. 3, 1)  GROUP 1           0.0
MESH (10. 3, 1)  GROUP 1          2.77556E-17
MESH (1. 4, 1)  GROUP 1           1.54363E-04
MESH (2. 4, 1)  GROUP 1           -1.26323E-04
MESH (3. 4, 1)  GROUP 1           -1.48187E-04
MESH (4. 4, 1)  GROUP 1           -2.60195E-05
MESH (5. 4, 1)  GROUP 1           3.15524E-04
MESH (6. 4, 1)  GROUP 1           3.16937E-04
MESH (7. 4, 1)  GROUP 1           -2.43373E-04
MESH (8. 4, 1)  GROUP 1           -1.38778E-16
MESH (9. 4, 1)  GROUP 1           8.32667E-17
MESH (10. 4, 1)  GROUP 1          1.38778E-17
MESH (1. 5, 1)  GROUP 1           -1.02292E-04
MESH (2. 5, 1)  GROUP 1           -2.93712E-04
MESH (3. 5, 1)  GROUP 1           -6.63648E-05
MESH (4. 5, 1)  GROUP 1           -1.77921E-05
MESH (5. 5, 1)  GROUP 1           2.58069E-04
MESH (6. 5, 1)  GROUP 1           -1.40196E-04
MESH (7. 5, 1)  GROUP 1           -8.18433E-05
MESH (8. 5, 1)  GROUP 1           1.11022E-16
MESH (9. 5, 1)  GROUP 1           -1.11022E-16
MESH (10. 5, 1)  GROUP 1          -4.16334E-17
MESH (1. 6, 1)  GROUP 1           -1.49587E-04
MESH (2. 6, 1)  GROUP 1           3.80838E-04
MESH (3. 6, 1)  GROUP 1           -1.91637E-04
MESH (4. 6, 1)  GROUP 1           3.04478E-04
MESH (5. 6, 1)  GROUP 1           4.24570E-05
MESH (6. 6, 1)  GROUP 1           1.94100E-04
MESH (7. 6, 1)  GROUP 1           2.26153E-04
MESH (8. 6, 1)  GROUP 1           5.55112E-17
MESH (9. 6, 1)  GROUP 1           5.55112E-17
MESH (10. 6, 1)  GROUP 1          4.16334E-17
MESH (1. 7, 1)  GROUP 1           -1.88228E-05
MESH (2. 7, 1)  GROUP 1           -3.61338E-05
MESH (3. 7, 1)  GROUP 1           2.59933E-05
MESH (4. 7, 1)  GROUP 1           3.09244E-04
MESH (5. 7, 1)  GROUP 1           -1.50483E-04
MESH (6. 7, 1)  GROUP 1           -2.77556E-17
MESH (7. 7, 1)  GROUP 1           2.77556E-17
MESH (8. 7, 1)  GROUP 1           2.77556E-17
MESH (9. 7, 1)  GROUP 1           2.77556E-17
MESH (10. 7, 1)  GROUP 1          -5.55112E-17
MESH (1. 8, 1)  GROUP 1           -9.44200E-05
MESH (2. 8, 1)  GROUP 1           2.21320E-04
MESH (3. 8, 1)  GROUP 1           -1.13433E-04
MESH (4. 8, 1)  GROUP 1           -1.48194E-04
MESH (5. 8, 1)  GROUP 1           1.52968E-04
MESH (6. 8, 1)  GROUP 1           -1.38778E-16
MESH (7. 8, 1)  GROUP 1           -8.32667E-17
MESH (8. 8, 1)  GROUP 1           -8.32667E-17
MESH (9. 8, 1)  GROUP 1           0.0
MESH (10. 8, 1)  GROUP 1          -8.32667E-17
MESH (1. 9, 1)  GROUP 1           2.18912E-05
MESH (2. 9, 1)  GROUP 1           1.61816E-04
MESH (3. 9, 1)  GROUP 1           1.59926E-04
MESH (4. 9, 1)  GROUP 1           -4.05419E-06
MESH (5. 9, 1)  GROUP 1           -1.13266E-04
MESH (6. 9, 1)  GROUP 1           5.55112E-17
MESH (7. 9, 1)  GROUP 1           8.32667E-17
MESH (8. 9, 1)  GROUP 1           1.38778E-16
MESH (9. 9, 1)  GROUP 1           1.38778E-17
MESH (10. 9, 1)  GROUP 1          0.0
MESH (1. 10, 1)  GROUP 1          1.62755E-04
MESH (2. 10, 1)  GROUP 1          -1.24794E-04
MESH (3. 10, 1)  GROUP 1          1.13449E-04
MESH (4. 10, 1)  GROUP 1          -1.86268E-04
MESH (5. 10, 1)  GROUP 1          2.74337E-05
MESH (6. 10, 1)  GROUP 1          -1.38778E-17
MESH (7. 10, 1)  GROUP 1          1.38778E-17
MESH (8. 10, 1)  GROUP 1          -2.77556E-17
MESH (9. 10, 1)  GROUP 1          -5.55112E-17
MESH (10. 10, 1)  GROUP 1         6.93889E-18
